win/rl/helper.cc

```diff
@@ -124,9 +124,18 @@ PYBIND11_MODULE(helper, m)
     m.def("glyph_is_warning",
           [](int glyph) { return glyph_is_warning(glyph); });
 
-    py::class_<permonst, std::unique_ptr<permonst, py::nodelete> >(m,
-                                                                   "permonst")
-        .def(py::init([](int index) -> permonst * { return &mons[index]; }))
+    py::class_<permonst>(m, "permonst")
+        .def(
+            "__init__",
+            [](py::detail::value_and_holder &v_h, int index) {
+                if (index < 0 || index >= NUMMONS)
+                    throw std::out_of_range(
+                        "Index should be between 0 and NUMMONS");
```

Comment:
should the error include what NUMMONS is, e.g. "Index should be between 0 and NUMMONS (int)"

Reply:
Kinda weird but okay.

Reply:
Ah I begin to understand what you meant. Fixed again.